### PR TITLE
W-11755770 fix links

### PR DESCRIPTION
**Pull request by IsaacEldridge**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by dmerlob — DISMISSED** [review]

File: modules/ROOT/pages/install-index.adoc [hunk 1]
```diff
@@ -3,5 +3,5 @@
 To install Runtime Fabric, choose from one of the following options:
 
 * xref:install-self-managed.adoc[Install GKE, AKS, EKS with rtfctl]
-* xref:install-helm.adoc[Install GKE, AKS, EKS with Helm]
-* xref:install-openshift[Install on RedHat OpenShift]
\ No newline at end of file
+* xref:install-helm.adoc.adoc[Install GKE, AKS, EKS with Helm]
```
Comment (cheungaryk) [inline modules/ROOT/pages/install-index.adoc]: there are 2 `.adoc`s? That doesn't look right to me - should be just `install-helm.adoc`

File: modules/ROOT/pages/upgrade-index.adoc [hunk 2]
```diff
@@ -3,5 +3,5 @@
 To upgrade Runtime Fabric, choose from one of the following options:
 
 * xref:upgrade-self-managed.adoc[Upgrade GKE, AKS, EKS with rtfctl]
-* xref:upgrade-helm.adoc[Upgrade GKE, AKS, EKS with Helm]
-* xref:upgrade-openshift[Upgrade RedHat OpenShift]
\ No newline at end of file
+* xref:upgrade-helm.adoc.adoc[Upgrade GKE, AKS, EKS with Helm]
```
Comment (cheungaryk) [inline modules/ROOT/pages/upgrade-index.adoc]: should be just `install-helm.adoc`

**Review by cheungaryk — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/install-index.adoc (hunk 1 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/install-index.adoc]:
```suggestion
* xref:install-helm.adoc[Install GKE, AKS, EKS with Helm]
```

File: modules/ROOT/pages/upgrade-index.adoc (hunk 2 above)
Comment (IsaacEldridge) [inline modules/ROOT/pages/upgrade-index.adoc]:
```suggestion
* xref:upgrade-helm.adoc[Upgrade GKE, AKS, EKS with Helm]
```

**Review by cheungaryk — APPROVED** [review]

**Pull request merged**